src/modules/billing/jobs/sync-charge-categories.js

```diff
@@ -11,7 +11,7 @@ const s3Connector = require('../../../lib/services/s3');
 
 // Constants
 const JOB_NAME = 'billing.charge-categories.sync-from-csv';
-const csvKey = 'billing-metadata/charge-categories.csv';
+const csvKey = 'billing-metadata/charge-categories-v1.csv';
```

Comment:
Why `-v1`?

Reply:
removed the -v1 was used for testing locally without upsetting dev